test/kubernetes/e2e/features/helm/types.go

```diff
@@ -18,4 +18,13 @@ var (
 			},
 		},
 	}
+
+	enterpriseCRDCategory = "solo-io"
```

Comment:
nit: It almost feels like these are a contract of our product, and deserve to live in our codebase, and then imported into our tests. It would also address my concern about enterprise crds expecting a solo-io category as being challenging for a new dev to discover

Reply:
Tbh I was a little surprised there wasn't already a list of the enterprise CRDs elsewhere in the code base, but I didn't find one. Any suggestions for where/in what format this ought to live?

Reply:
there is a list in enterprise [here](https://github.com/solo-io/solo-projects/blob/257a684fabde7388874934f930ef0da81ad0ebfc/projects/gloo/pkg/setup/history.go#L18). i don't know if it makes sense to add any more enterprise-specific code in our OSS repo (although the enterprise crds live in the oss repo so it's a little messy to begin with). maybe it makes sense to add these new enterprise-specific tests in solo-projects only?